content/blog/2018/06/2018-05-14-simple-pull-request-plugin.adoc

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: "GSoC Project Intro: Simple pull request Plugin"
+tags:
+- gsoc2018
+- plugin
+author: abhishek_gautam
+---
+
+=== About me
+
+I am Abhishek Gautam, 3rd year student from Visvesvaraya National Institute of
+technology, India, Nagpur. I was a member of ACM Chapter and Google student developer club of his
+college. I am passianate about automation.
+
+=== Mentors
+
+* link:https://github.com/oleg-nenashev[Oleg Nenashev] (Org Admin)
+* link:https://github.com/martinda[Martin d'Anjou]
+* link:https://github.com/kwhetstone[Kristin Whetstone]
+* link:https://github.com/grandvizier[Jeff Knurek]
+
+=== Project Summary
+
+This is a GSoC 2018 project.
+
+This project aims to develop a Pull request Job Plugin. Users should be able to
+configure job type using YAML file which will be placed in root directory of the
+Git repository being the subject of the PR. Plugin should interact with various
+platfoms like Bitbucket, Github, Gitlab, etc whenever a pull request is created or updated.
+
+Plugin should be able to detect the presence of certain types of the report at conventional locations,
```

Comment:
The plugin detects the presence of certain types of reports at conventional locations, ...

Reply:
This is a section explaining, what this plugin should do. Hence I think statements should be like a wish. Should I change it anyway ?

Reply:
You'll cover that in the "what's available now" section.  Let's treat this section as a description of the plugin's goals.